### DISCUSSION TITLE
3950x and sx8200 2tb with ramdisk

### DISCUSSION THREAD

**Discussion by hiromasa0629** [madMAx43v3r/chia-plotter#657] [Q&A]
Specs: 
3950x 4.1ghz
4x32gb 2400mhz (created 111gb ramdisk)
sx8200 2tb

Set it to 16thread because 10gb of ram is available after ramdisk.
![image](https://user-images.githubusercontent.com/81148313/123186656-31917d00-d4cb-11eb-8ee7-dd58fce5e2ce.jpeg)

40mins with ddr4 ram and 16threads. Plots completion 40mins. Isnt this abit slow ? How do i improve ?

**Comment by CoolNuker** [comment]
Ubuntu will make it within1100sec. Forget you windows sys

**Reply by vladaepro** [reply]
Clear Linux even faster
